Source/RMG/UserInterface/Widget/RomBrowserWidget.cpp

```diff
@@ -460,15 +448,27 @@ void RomBrowserWidget::customContextMenuRequested(QPoint position)
 
     this->getCurrentData(data);
 
+    this->mainContextMenu->clear();
```

Comment:
Like I mentioned above, please revert this

Reply:
Done, will include in next commit.